src/codegen.cpp

```diff
@@ -4723,7 +4719,7 @@ static std::unique_ptr<Module> emit_function(jl_lambda_info_t *lam, jl_llvm_func
                                              0,
                                              true,
                                              nullptr);
-                builder.SetCurrentDebugLocation(DebugLoc::get(0, 0, (MDNode*)SP, builder.getCurrentDebugLocation()));
+                builder.SetCurrentDebugLocation(DebugLoc::get(inlfline, 0, (MDNode*)SP, builder.getCurrentDebugLocation()));
             }
```

Comment:
I don't understand --- surely we don't want the line of the first statement to actually be reported as zero. Should I do something like calling `SetCurrentDebugLocation` with 0, followed immediately by calling it again with the actual number?